Makefile

```diff
@@ -0,0 +1,36 @@
+# ./node_modules/.bin on the PATH
+export PATH := ./node_modules/.bin:$(PATH)
+
+# Use bash not sh
+SHELL := /bin/bash
+
+# Environment variables
+# ---------------------
+
+EXPECTED_COVERAGE = 50
```

Comment:
We should look to improve this coverage in another PR

Reply:
https://github.com/Financial-Times/get-github-public-organisation-repositories/issues/4